docs/pages/getting-started/linux-server.mdx

```diff
@@ -166,12 +166,26 @@ On your Linux machine, run the following command to start the `teleport` daemon
   </TabItem>
 
   <TabItem label="Source or custom install">
+  1. Create a systemd unit file using the `teleport install systemd` command and the [unit file path](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) you'd
```

Comment:
I recommend moving additional articles and links to the last section "Next steps" to avoid [breaking readers focus](https://goteleport.com/docs/contributing/documentation/style-guide/#expectations) according to our style guide.

Reply:
Moved all links to the "Next Steps" section, retitled from "Further Reading" to better fit our style guide